egs/jsut/asr1/local/make_pitch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#This script is to adjust sampling rate of the wav files and config files.
+
+pitch=$1
+
+# fix fbank.conf
+sed -i -e "s/sample-frequency=[0-9]*/sample-frequency=$pitch/" ./conf/fbank.conf
```

Comment:
This is a little bit risky way. My suggestion
1) prepare `conf/fbank_16k.conf` and `conf/pitch_16k.conf`, and git add them.
2) remove `conf/fbank.conf` and `conf/pitch.conf` by git rm
3) this script generate `./conf/fbank.conf` from `./conf/fbank_16k.conf
```
sed -e "s/sample-frequency=[0-9]*/sample-frequency=$pitch/" ./conf/fbank_16k.conf > ./conf/fbank.conf
```

Reply:
Okay!